v4/source/roles.yaml

```diff
@@ -0,0 +1,150 @@
+  '/roles/name/{role_id}':
+    get:
+      tags:
+        - roles
+      summary: Get a role
+      description: |
+        Get a role from the provided role id.
+        ##### Permissions
+        Requires an active session but no other permissions..
+      parameters:
+        - name: role_id
+          in: path
+          description: Role GUID
+          required: true
+          type: string
+      responses:
+        '200':
+          description: Role retrieval successful
+          schema:
+            $ref: '#/definitions/Role'
+        '401':
+          $ref: '#/responses/Unauthorized'
+        '404':
+          $ref: '#/responses/NotFound'
+      x-code-samples:
+        - lang: 'Go'
+          source: |
+            import "github.com/mattermost/mattermost-server/model"
+            Client := model.NewAPIv4Client("https://your-mattermost-url.com")
+            Client.Login("email@domain.com", "Password1")
+
+            // GetChannel
```

Comment:
Comment is wrong here (and for the other examples), can probably just be removed